generated/api.github.com.json

```diff
@@ -324,42 +356,8 @@
         "POST /repos/{owner}/{repo}/deployments/{deployment_id}/statuses"
       ]
     },
-    "emails": {
-      "url": "https://docs.github.com/en/free-pro-team@latest/rest/reference/permissions-required-for-github-apps/#permission-on-emails",
-      "read": ["GET /user/emails", "GET /user/public_emails"],
-      "write": [
-        "DELETE /user/emails",
-        "PATCH /user/email/visibility",
-        "POST /user/emails"
-      ]
-    },
-    "followers": {
-      "url": "https://docs.github.com/en/free-pro-team@latest/rest/reference/permissions-required-for-github-apps/#permission-on-followers",
-      "read": [
-        "GET /user/followers",
-        "GET /user/following",
-        "GET /user/following/{username}"
-      ],
-      "write": [
-        "DELETE /user/following/{username}",
-        "PUT /user/following/{username}"
-      ]
-    },
-    "gpg_keys": {
-      "url": "https://docs.github.com/en/free-pro-team@latest/rest/reference/permissions-required-for-github-apps/#permission-on-gpg-keys",
-      "read": ["GET /user/gpg_keys", "GET /user/gpg_keys/{gpg_key_id}"],
-      "write": ["DELETE /user/gpg_keys/{gpg_key_id}", "POST /user/gpg_keys"]
-    },
-    "interaction_limits": {
-      "url": "https://docs.github.com/en/free-pro-team@latest/rest/reference/permissions-required-for-github-apps/#permission-on-interaction-limits",
-      "read": ["GET /user/interaction-limits"],
-      "write": [
-        "DELETE /user/interaction-limits",
-        "PUT /user/interaction-limits"
-      ]
-    },
```

Comment:
removes all the above because they are not defined in OpenAPI schema and I'm not sure if this is the correct name